CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## 1.0.0
+
+Initial Stable Release
+
+## 1.0.1
+
+* Display additional features in the [docs.rs/tinyvec](https://docs.rs/tinyvec) documentation
+
+## 1.0.2
```

Comment:
I think it would be better if the changelog had versions in the opposite order, so newest releases first. Most users will want to see whats changed from their current version to the latest, in the case of updating a dependency, looking for a behavior change they may of missed, etc. Imo a top down read gives the best experience for cases like that.